src/models/SearchResultsFilters.ts

```diff
@@ -11,6 +11,7 @@ class SearchResultsFilters {
   constructor(aggregationsResults: Aggregation[], field: Option) {
     this.labelTransformations = {
       "Greek, Modern (1453- )": "Greek, Modern (1453-present)",
+      Offsite: "Offsite - (Deliverable to NYPL research libraries)",
```

Comment:
Ah yes, this lonely outlier.

Reply:
I was actually really pleased to finally have another label transformation to extend this with!